### ISSUE TITLE
QUESTION: Is there an option to wrap the text in the Query Editor tabs?

### ISSUE THREAD

**Issue by eben-flow** [beekeeper-studio/beekeeper-studio#2320]
I'm coming from Tableplus which automatically wraps queries so that I can avoid horizontal scrolling. I want to know if this is present in beekeeper as well because I'm not able to find it.

**Comment by EmanNo1** [comment]
Have you tried "format query" on the right-click pop-up menu?

**Comment by eben-flow** [comment]
Yes, that works but that is not exactly what I'm looking for. I would like to have a normal wrap text feature that's there is most text editors so that the content of a line drops down like a new line as soon as the line's length exceeds the display/window width.
That way I also have some control over how the queries are formatted.

**Comment by not-night-but** [comment]
Yeah we should definitely add an option to wrap text, this has been bugging me as well (particularly in the json-view).

**Comment by skogsmaskin** [comment]
It's a dealbreaker for me that you can't properly preview that data when you select a row. A truncated JSON view is not what I consider an acceptable view for humans to look at data in. Or am I doing this wrong? I want to be able to select a row and easily read the text that's in the fields.

It's a shame because the rest of the app seems awesome.

I'm also coming from Table Plus btw.
